### ISSUE TITLE
[WINDOWS, ANDROID] FlyoutIsPresented is not working as expected on Android and Windows when navigating between FlyoutItems

### ISSUE THREAD

**Issue by NafeelaNazhir** [dotnet/maui#26422]
### Description

When navigating between FlyoutItems in a Shell application, setting FlyoutIsPresented = true in the Shell.Navigating event does not keep the flyout open on Android and Windows platforms. The flyout automatically closes despite the property being explicitly set.
https://github.com/user-attachments/assets/14d0bc2b-58af-4c42-9496-6c3d75212543

### Steps to Reproduce

1.Run the Application [Issue11247Sample.zip](https://github.com/user-attachments/files/18036293/Issue11247Sample.zip)
2.Open the Flyout  and Tap "FlyoutItem 2"
3.Observe that Flyout should remains open.
Expected Behavior:
The flyout should remain open, when navigating between FlyoutItems.

Actual Behavior:
The flyout closes immediately after tapping on the FlyoutItem.

### Version with bug

9.0.12 SR1.2

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Affected platforms

Android, Windows

### Relevant log output

```shell

```

**Comment by Ying-6** [comment]
This issue has been verified using Visual Studio 17.13.0 Preview 1.0 (9.0.12 & 9.0.10 & 8.0.100). Can repro on windows and Android platforms.

**Comment by jfversluis** [comment]
Duplicate of #26324

**Comment by NafeelaNazhir** [comment]
The issue was marked as a duplicate of a referenced issue, but the two scenarios are distinct and require separate resolutions. The referenced issue addresses the flyout closing automatically when the FlyoutPage.Detail is updated. However, the current issue focuses on the general behavior of the FlyoutIsPresented property, which is unrelated to detail updates.

**Comment by jfversluis** [comment]
If you tap a flyout item the Detail is updated, so I do think this is a duplicate.

Additionally, I wonder if this is even actually a bug? Isn't the default behavior of this control to select an item and show it? I don't really see the use-case where someone would tap an option in the flyout and it should remain open?

**Comment by kubaflo** [comment]
I agree with @jfversluis. And it is actually the native Android behavior, so I doubt it is a bug. Additionally, I think that if developers want they can set FlyoutIsPresented to true after a user selects an option from the flyout menu to avoid flyout being hidden

The app from Android Studio template:
<video src="https://github.com/user-attachments/assets/04789a86-4425-43ae-8239-95ee13e79ac2"/>

**Comment by kubaflo** [comment]
Closing as a duplicate of #26324. Approve. A maintainer explicitly marked #26324 as the duplicate; the target is a valid completed issue covering the same FlyoutPage detail-update behavior.

**Comment by kubaflo** [comment]
Reopening — this issue should not have been closed.

It carries `s/verified` (a maintainer or partner reproduced it), `s/triaged` (it already went through triage), `partner/syncfusion` (a partner team is tracking it). Those labels mean the report was already confirmed as a real product defect, so it is not eligible for backlog cleanup closure. Closure here requires specific fix evidence (a merged PR that closes it, or a `fixed-in-*` label), and there was none.

I have also removed the `s/try-latest-version` label that this cleanup pass added, so the policy bot will not auto-close it again.

Sorry for the noise, and thanks to everyone who flagged it.

> Disclosure: this comment was generated by AI.
